modules/nf-core/hiphase/meta.yml

```diff
@@ -0,0 +1,94 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hiphase"
+description: write your description here
+keywords:
+  - sort
+  - example
+  - genomics
+tools:
+  - "hiphase":
+      description: "Small and structural variant phasing tool for PacBio HiFi reads"
+      homepage: "https://github.com/PacificBiosciences/HiPhase"
+      documentation: "https://github.com/PacificBiosciences/HiPhase"
+      tool_dev_url: "https://github.com/PacificBiosciences/HiPhase"
+      doi: "10.1234/placeholder.doi"
+      licence: ["BSD-3-clause-Clear"]
```

Comment:
The newer [licence](https://github.com/PacificBiosciences/HiPhase/blob/main/LICENSE.md) for HiPhase is a bit tricky, I don't think it's BSD-3-clause-Clear anymore. For example: 

> License Restrictions. You may not remove or destroy any copyright notices or other proprietary markings. **You may only use the Software to process or analyze data generated on a PacBio instrument or otherwise provided to you by PacBio**. [...] You may not use, modify, host, or distribute the Software so that any part of the Software becomes subject to any license that requires, as a condition of use, modification, hosting, or distribution, that (a) the Software, in whole or in part, be disclosed or distributed in source code form or (b) any third party have the right to modify the Software, in whole or in part.

Reply:
I see - I missed this, thanks for pointing it out!

Reply:
What would be the best way to handle this? - Can I still create a module for the software?

Reply:
Yeah, it's bit unfortunate. I'm not sure, I posted a question on slack: https://nfcore.slack.com/archives/C043UU89KKQ/p1730099874046369. 

It would be nice to have it in nf-core.

Reply:
What about "Pacific Biosciences Software License" ?

Do you think it's fine to use the conda version, or should we use the pacbio docker?